sdk/anomalydetector/Azure.AI.AnomalyDetector/src/autorest.md

```diff
@@ -0,0 +1,9 @@
+# Generated code configuration
+
+Run `dotnet msbuild /t:GenerateCode` to generate code.
+
+``` yaml
+input-file:
+    -  https://github.com/Azure/azure-rest-api-specs/blob/1280dd676549a0631330945508bcee05ee93c800/specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json
+namespace: Azure.AI.AnomalyDetector
```

Comment:
enabling `public-clients: true` here would allow more code to be generated.

Reply:
`public-clients: true` will generate `ServiceClient`. So we will add more constructor in this file only inside generated folder? and rename it to `AnomalyDetectorClient` instead of manually creating https://github.com/Azure/azure-sdk-for-net/pull/14293/files#diff-8130b8b77496917a035245907dfda473 file?

Reply:
>So we will add more constructors in this file only inside generated folder? 

You can add constructors in a partial class outside of the generated folder.

> and rename it to AnomalyDetectorClient instead of manually creating #14293 (files) fil
Yes